src/component/Note/Note.jsx

```diff
@@ -0,0 +1,108 @@
+import "./note.css";
+import { useState } from "react";
+import { useNote } from "../../context/NoteContext";
+
+const AddNote = () => {
+  const { addNotes } = useNote();
+  const [isNoteVisible, setIsNoteVisisble] = useState(false);
+  const initialState = {
```

Comment:
store the initialState into a separte file and then import and use

Reply:
ok will do that